html.ddoc

```diff
@@ -435,4 +435,5 @@ HEARTS = &hearts;
 DIAMS = &diams;
 LANG = &lang;
 RANG = &rang;
+TAB = &#09;
```

Comment:
This should probably not be at the end of this list though, which seems to track another one (https://dlang.org/entity.html).

Reply:
-> https://github.com/dlang/dlang.org/pull/2297